web-client/src/components/LanguageSelect/index.tsx

```diff
@@ -1,29 +1,30 @@
-import { useState, CSSProperties } from 'react';
+import { CSSProperties } from 'react';
 import InputLabel from '@mui/material/InputLabel';
 import MenuItem from '@mui/material/MenuItem';
 import FormControl from '@mui/material/FormControl';
 import Select, { SelectChangeEvent } from '@mui/material/Select';
 import Box from '@mui/material/Box';
+import {useSettingsStore}  from 'store/useSettings'
 
 interface LangSelectProps {
     style: CSSProperties;
   }
 const LangSelect: React.FC<LangSelectProps> = ({ style }) =>{
-    const [lang, setLang] = useState("ua");
-    const handleChange = (event: SelectChangeEvent) => {
-        setLang(event.target.value as string);
-    };
+    const { isMobile, language, setLanguage } = useSettingsStore();
+
     return(
-        <Box sx={{ width: '100px', height: '40px' }}>
+        <Box sx={{width: isMobile ? '100px' : '140px', height: '40px'}}>
```

Comment:
is it possible to use something like this: https://mui.com/material-ui/customization/breakpoints/?

Reply:
To utilize the framework and react consitently then we would not need the isMobile state in useSettingsStore

Reply:
1.I have changed materialUI Box to custom component FlexBox;
2.I have deleted isMobile from the state in useSettingsStore and created custom hook useMobileSize